src/studio-home/tabs-section/TabsSection.test.tsx

```diff
@@ -347,34 +323,33 @@ describe('<TabsSection />', () => {
   });
 
   describe('library tab', () => {
+    beforeEach(() => {
+      axiosMock.onGet(courseApiLink).reply(200, generateGetStudioCoursesApiResponse());
+    });
     it('should switch to Legacy Libraries tab and render specific v1 library details', async () => {
-      render(<RootWrapper />);
+      render();
       axiosMock.onGet(getStudioHomeApiUrl()).reply(200, generateGetStudioHomeDataApiResponse());
       axiosMock.onGet(libraryApiLink).reply(200, generateGetStudioHomeLibrariesApiResponse());
       await executeThunk(fetchStudioHomeData(), store.dispatch);
       await executeThunk(fetchLibraryData(), store.dispatch);
 
       const librariesTab = screen.getByText(tabMessages.legacyLibrariesTabTitle.defaultMessage);
-      await act(async () => {
-        fireEvent.click(librariesTab);
-      });
+      fireEvent.click(librariesTab);
 
       expect(librariesTab).toHaveClass('active');
 
-      expect(screen.getByText(studioHomeMock.libraries[0].displayName)).toBeVisible();
+      expect(await screen.findByText(studioHomeMock.libraries[0].displayName)).toBeVisible();
 
       expect(screen.getByText(`${studioHomeMock.libraries[0].org} / ${studioHomeMock.libraries[0].number}`)).toBeVisible();
     });
 
     it('should switch to Libraries tab and render specific v2 library details', async () => {
-      render(<RootWrapper />);
+      render();
       axiosMock.onGet(getStudioHomeApiUrl()).reply(200, generateGetStudioHomeDataApiResponse());
       await executeThunk(fetchStudioHomeData(), store.dispatch);
 
       const librariesTab = screen.getByText(tabMessages.librariesTabTitle.defaultMessage);
-      await act(async () => {
-        fireEvent.click(librariesTab);
-      });
+      fireEvent.click(librariesTab);
```

Comment:
Note: `act()` is not necesary - `testing-library`'s `fireEvent` method automatically wraps the event with `act` for you, and they say you never need to write `act` when using `testing-library`. If you see warnings about needing `act()`, it's because some async thing is updating the component unexpectedly, and it can usually be fixed using `waitFor`.